gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=0.2.16-SNAPSHOT
```

Comment:
When this is merged we will create some jobs on cloudbees. These jobs will
let you specify if you want to bump the major minor or patch version
number. I can talk to you in more detail by showing you some of the
existing jobs.

I think what I deleted still works. Our provided plugin is setup to be
added to many configs. I would definitely run some tests and take a look to
make sure I didn't break anything though.

On Wed, Mar 18, 2015 at 9:26 AM, Matt Bossenbroek <notifications@github.com>
wrote:

> In gradle.properties
> <https://github.com/Netflix/PigPen/pull/146#discussion_r26679857>:
>
> > @@ -1,2 +1,2 @@
> > -version=0.2.16-SNAPSHOT
>
> How would I bump the version to 0.3.0 now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Netflix/PigPen/pull/146/files#r26679857>.
>